### PR TITLE
No Cookies

### PR DESCRIPTION
**Pull request by alexandergull**

### CODE REVIEW

File: cleantalk.php [hunk 1]
```diff
@@ -626,6 +626,8 @@ function apbct_activation__new_blog($blog_id, $_user_id, $_domain, $_path, $_sit
     require_once(CLEANTALK_PLUGIN_DIR . 'inc/cleantalk-public.php');
     require_once(CLEANTALK_PLUGIN_DIR . 'inc/cleantalk-public-integrations.php');
 
+    apbct_form__get_noCookie_data();
```
Comment (ArtemAnoshin) [inline cleantalk.php:629]: need a comment. mixing camelcase and snake_case. there is a word to get, but the result does not return anywhere.

**Review by ArtemAnoshin — COMMENTED** [review]
The remark is rhetorical, you can ignore it, but I decided to share my opinion)

File: cleantalk.php (hunk 1 above)
Comment (Glomberg) [inline cleantalk.php:629]: Optimization: Thes function will calls update_option on every hit. Need to be reduced the update_option calling.

File: cleantalk.php [hunk 2]
```diff
@@ -2190,7 +2192,7 @@ function apbct_store__urls()
     global $apbct;
 
     if (
-        $apbct->data['cookies_type'] === 'none' || // Do not set cookies if option is disabled (for Varnish cache).
+        //$apbct->data['cookies_type'] === 'none' || // Do not set cookies if option is disabled (for Varnish cache).
```
Comment (Glomberg) [inline cleantalk.php:2195]: Do not afraid to remove unused code, not just comment it.

File: cleantalk.php [hunk 3]
```diff
@@ -2254,7 +2256,7 @@ function apbct_cookie()
...
```
Comment (Glomberg) [inline cleantalk.php:2259]: Do not afraid to remove unused code, not just comment it.

File: js/src/apbct-public--1--functions.js [hunk 4]
```diff
@@ -208,5 +210,19 @@ apbctLocalStorage = {
...
-    }
+    },
+    getCleanTalkData : function () {
+        let data = {}
+        for(let i=0; i<localStorage.length; i++) {
+            let key = localStorage.key(i);
+
+            if (key.indexOf('ct_') !==-1 || key.indexOf('apbct_') !==-1){
+                // console.log(key)
+                // console.log(apbctLocalStorage.get(key))
```
Comment (Glomberg) [inline js/src/apbct-public--1--functions.js:221]: Please remove the debugging code.

**Review by Glomberg — APPROVED** [review]
We can to release this functionality as is but only after manual testing success. Dut we should to do some work about the optimisation.

File: cleantalk.php (hunk 2 above)
Comment (alexandergull) [reply cleantalk.php:2195]: Done

File: cleantalk.php (hunk 3 above)
Comment (alexandergull) [reply cleantalk.php:2259]: Done

File: js/src/apbct-public--1--functions.js (hunk 4 above)
Comment (alexandergull) [reply js/src/apbct-public--1--functions.js:221]: Done

File: cleantalk.php (hunk 1 above)
Comment (alexandergull) [reply cleantalk.php]: Done.

**Pull request merged**